### DISCUSSION TITLE
KDE packages are out of date

### DISCUSSION THREAD

**Discussion by ppvnf** [ScoopInstaller/Extras#9749] [General]
Issue:

https://github.com/ScoopInstaller/Extras/issues/9289

Discussion:

https://github.com/ScoopInstaller/Extras/pull/9298
https://github.com/ScoopInstaller/Extras/pull/9514

New version:

https://github.com/joaoricarte/jr-bucket/blob/main/bucket/okular.json
https://github.com/joaoricarte/jr-bucket/blob/main/bucket/kate.json

**Comment by StarsbySea** [comment]
https://github.com/ScoopInstaller/Extras/blob/c42b17b053aace52e7bebeb34b85978b8c8ad130/bucket/okular.json
https://github.com/ScoopInstaller/Extras/blob/0332d430473b89ab3cb8ddb82e7c2166e5f16285/bucket/kate.json
This has been fixed, have a quick look at the link above.

**Reply by ppvnf** [reply]
That's good to know. Thanks!
